src/lesson6tasks/Point.java

```diff
@@ -0,0 +1,14 @@
+package lesson6tasks;
+
+public class Point {
+    private double x;
+    private double y;
+    public Point (double x, double y) {
+        this.x = x;
+        this.y = y;
+    }
+    public static double calculateDistance (Point first, Point second) {
+        return Math.sqrt((first.x - second.x) * (first.x - second.x) +
+                (first.y - second.y) * (first.y - second.y));
```

Comment:
відстань повинна рахуватися від обєкта this до обєкта який передається в параметрах 
```suggestion
    public static double calculateDistance (Point first) {
        return Math.sqrt(Math.pow((this.x - first.x), 2) + Math.pow((this.y - first.y), 2));
```